docs/cli/ctl.md

```diff
@@ -0,0 +1,369 @@
+---
+title: "wash ctl"
+draft: false
+sidebar_position: 6
+description: "wash ctl command reference"
+--- 
+
+This subcommand will help you to interact with the wasmCloud control interface. You may perform operations such as retrieving information about the lattice, enable provisioning or deprovisioning of entities in the lattice, creating links or updating items in the lattice. Following are the subcommands available under `wash ctl`
+
+- `get`
+- `link`
+- `start`
+- `stop`
+- `update`
+- `scale`
+
+## `get`
+wash get retrives information about the lattice. The following subcommands will help you retrieve different information available to query:
+
+:::caution
+This subcommand will be deprecated in furture versions. Please use `wash get` instead.
```

Comment:
Looks like the `furture` fat finger was copy pasted everywhere. I'd do a quick check across all the docs here